### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by kyleGrealis**
- updated gist path for `.gitignore`
- silenced console output for downloading files and returning "[1] TRUE" when creating projects
- added `README.md` template for projects
- moved `@importFrom` to `R/rUM-package.R` to centralize dependencies
- added `write_scss()` function
- added `write_scss()` inside `make_project()` to add during project creation

### CODE REVIEW

**Comment by kyleGrealis** [comment]
- updated PR to remove external gist downloads ([issue #57](https://github.com/RaymondBalise/rUM/issues/57))
- added DATED_PROGRESS_NOTES.md template & written to .Rbuildignore
- passed `check()` with no warnings, notes, errors

**Comment by kyleGrealis** [comment]
needed to modify my original downloading of package internal gists to use `system.file()` or the package fails to write these to new projects

File: R/write_scss.R [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#' Create a Quarto SCSS file
+#' 
+#' This function creates the \code{.scss} file so that any Quarto project can be easily 
+#' customized with SCSS styling variables, mixins, and rules.
+#' 
+#' For more information on customizing Quarto documents with SCSS, please refer to
+#' \url{https://quarto.org/docs/output-formats/html-themes.html#customizing-themes} and
+#' \url{https://quarto.org/docs/output-formats/html-themes-more.html}. If you really
+#' would like to go down a rabbit hole:
```
Comment (RaymondBalise) [inline R/write_scss.R:9]: This will not translate well for non-english speakers

File: inst/old/analysis_1_0_2_9001.Rmd
Comment (RaymondBalise) [inline inst/old/analysis_1_0_2_9001.Rmd:1]: I am not sure the _9001 files are the latest.  We should do a diff

**Review by RaymondBalise — CHANGES_REQUESTED** [review]

**Review by RaymondBalise — CHANGES_REQUESTED** [review]
Update the news using my note

**Review by RaymondBalise — APPROVED** [review]

**Pull request merged**